src/cli/index.js

```diff
@@ -64,14 +65,26 @@ async function runTests (argParser) {
         selectorTimeout: opts.selectorTimeout
     });
 
+    await testCafe.close();
+
     exit(failed);
 }
 
-async function listBrowsers () {
-    var installations = await getBrowserInstallations();
-    var aliases       = Object.keys(installations);
+async function listBrowsers (providerName = 'local') {
+    var provider = await browserProviderPool.getProvider(providerName);
+
+    if (!provider)
+        throw new GeneralError(MESSAGE.browserProviderNotFound, providerName);
+
+    var aliases = await provider.listAvailableBrowserNames();
+
+    await browserProviderPool.dispose();
+
+    if (!aliases.length)
+        console.log(`"${providerName}"`);
+    else
+        console.log(aliases.map(alias => `"${providerName}:${alias}"`).join('\n'));
```

Comment:
For local provider we shouldn't print `${providerName}:${alias}`, we should use `${alias}` only. Also, I've asked to reflect in code structure that alias is providerName+browserName.